covfee/client/admin/hit_block/node_buttons.tsx

```diff
@@ -46,6 +46,7 @@ export const NodeRow = ({
           <StatusIcon color={NodeStatusToColor[getNodeStatus(node)]} />
         </span>
         {node.name}[{node.id}] - {node.status}
+        {node.progress ? "(" + node.progress + "%)" : ""}
```

Comment:
Do we want to do extra formatting here to show only two decimals or something like that?